src/modules/routing/components/waypoints/Waypoints.js

```diff
@@ -89,13 +88,8 @@ export class Waypoints extends MvuElement {
 
 		return waypoints.length > 0
 			? html`<div class="waypoints__actions">
-					<ba-icon
-						id="button_reverse"
-						.icon="${arrowDownUpSvg}"
-						.size=${1.5}
-						.title=${translate('routing_waypoints_reverse')}
-						@click=${reverse}
-					></ba-icon>
+					<ba-button id="button_reverse" .label=${translate('routing_waypoints_reverse')} @click=${reverse}></ba-button>
+					<ba-button id="button_clear" .label=${translate('routing_waypoints_remove_all')} @click=${reset}></ba-button>
```

Comment:
There should be a test here that checks whether the interface is re-rendered accordingly after the "reset"-click

Reply:
done